frontend/frontend.iml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the purpose of this xml? Can we remove it?

Reply:
idea settings, removed them